xbmc/pvr/timers/PVRTimers.cpp

```diff
@@ -469,9 +469,9 @@ bool CPVRTimers::InstantTimer(const CPVRChannel &channel)
   if (!g_PVRManager.CheckParentalLock(channel))
     return false;
 
-  CEpgInfoTag epgTag;
-  bool bHasEpgNow = channel.GetEPGNow(epgTag);
-  CPVRTimerInfoTag *newTimer = bHasEpgNow ? CPVRTimerInfoTag::CreateFromEpg(epgTag) : NULL;
+  CEpgInfoTagPtr epgTag(channel.GetEPGNow());
+  bool bHasEpgNow = (epgTag != NULL);
```

Comment:
@opdenkamp Thought, that <code>bool operator!=(shared_ptr<T> const & p, std::nullptr_t);<code> would handle that correctly? Am I wrong?